diff_settings.py

```diff
@@ -3,3 +3,7 @@ def apply(config, args):
     config["myimg"] = "source.bin"
     config["mapfile"] = "build.map"
     config["source_directories"] = ["."]
+    #config['arch'] = 'mips'
+    #config['map_format'] = 'gnu' # gnu or mw
+    #config['makeflags'] = []
+    #config['objdump_executable'] = ''
```

Comment:
```suggestion
    #config["arch"] = "mips"
    #config["map_format"] = "gnu" # gnu or mw
    #config["makeflags"] = []
    #config["objdump_executable"] = ""
```
(for `black` compliance; I'm planning to reformat the repo at some point)